custom/icds/translations/integrations/client.py

```diff
@@ -40,10 +40,17 @@ def get_resource_slugs(self, version):
         """
         all_resources = self.list_resources().json()
         if version and self.use_version_postfix:
+            # get all slugs with version postfix
             return [r['slug']
                     for r in self.list_resources().json()
                     if r['slug'].endswith("v%s" % version)]
+        elif version and not self.use_version_postfix:
+            # get all slugs that don't have version postfix
+            return [r['slug']
+                    for r in self.list_resources().json()
```

Comment:
nit: you can use `all_resources` here and on line 45 rather than `self.list_resources().json()`